jsonnet/kube-thanos/kube-thanos-store.libsonnet

```diff
@@ -22,6 +22,9 @@ function(params) {
 
   // Safety checks for combined config of defaults and params
   assert std.isNumber(ts.config.replicas) && ts.config.replicas >= 0 : 'thanos store replicas has to be number >= 0',
+  assert std.isNumber(ts.config.limits.seriesTouched) && ts.config.limits.seriesTouched >= 0 : 'thanos store series touched limit has to be number >= 0',
```

Comment:
Let's maybe check if this is object as well?
```suggestion
  assert std.isObject(ts.config.limits) : 'thanos store series limit has be an object',
  assert std.isNumber(ts.config.limits.seriesTouched) && ts.config.limits.seriesTouched >= 0 : 'thanos store series touched limit has to be number >= 0',
```

Reply:
Good idea, thanks! Will add this.

Reply:
Done in https://github.com/thanos-io/kube-thanos/pull/292/commits/adf2f1c480eae000e83da97a31850c2580595c2e.